src/main/java/me/xemor/yaggaskabble/Alignment.java

```diff
@@ -2,10 +2,10 @@
 
 public enum Alignment {
 
-    GOOD,EVIL;
+    GOOD,EVIL,BOTH;
```

Comment:
I don't think this is a good idea as the alignment enum is used for more than just the command.